datafusion/sqllogictest/test_files/string_view.slt

```diff
@@ -895,14 +895,13 @@ logical_plan
 02)--TableScan: test projection=[column1_utf8view, column2_utf8view]
 
 ## Ensure no casts for TRANSLATE
-## TODO file ticket
 query TT
 EXPLAIN SELECT
   TRANSLATE(column1_utf8view, 'foo', 'bar') as c
 FROM test;
 ----
 logical_plan
-01)Projection: translate(CAST(test.column1_utf8view AS Utf8), Utf8("foo"), Utf8("bar")) AS c
+01)Projection: translate(test.column1_utf8view, Utf8("foo"), Utf8("bar")) AS c
```

Comment:
I think we should probably add a test that actually invokes the function with a `StringVieAwArray` argument.

For example:

```
SELECT
  TRANSLATE(column1_utf8view, 'foo', 'bar') as c
```